core/src/main/java/com/linecorp/armeria/server/docs/EnumValueInfo.java

```diff
@@ -32,24 +32,37 @@
     private final String name;
     @Nullable
     private final String docString;
+    private final String literalValue;
```

Comment:
How about annotating this field with `@Nullable` and allowing `null` to represent the case where the literal value is not available?

Reply:
Let's add this field to equals, hashCode, and toString. This will cause some tests to need to be updated, which is good since it means the new code is tested :)